src/others/snapshot/lv_snapshot.h

```diff
@@ -63,8 +93,22 @@ uint32_t lv_snapshot_buf_size_needed(lv_obj_t * obj, lv_color_format_t cf);
  * @param buf_size  provided buffer size in bytes.
  * @return          LV_RESULT_OK on success, LV_RESULT_INVALID on error.
  */
-lv_result_t lv_snapshot_take_to_buf(lv_obj_t * obj, lv_color_format_t cf, lv_image_dsc_t * dsc, void * buf,
-                                    uint32_t buf_size);
+static inline lv_result_t _lv_snapshot_take_to_buf(lv_obj_t * obj, lv_color_format_t cf, lv_image_dsc_t * dsc,
```

Comment:
Should we remove the legacy API directly? This usage should be forbidden.

Reply:
I think it's enough provide an example in the docs.

Reply:
Since the new API name is different, let keep the legacy API here.